### PR TITLE
Chunk shape

### PR DESCRIPTION
**Pull request by normanrz**
### Description:
- Renames `chunk_size` -> `chunk_shape` and `shard_size` -> `shard_shape`, while keeping the old names in place for backwards compatibility.
- Adds wkcuber support for webdav URLs

### Issues:
- fixes https://github.com/scalableminds/webknossos/issues/6053

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [x] Updated Changelog

### CODE REVIEW

File: wkcuber/wkcuber/_internal/utils.py [hunk 1]
```diff
@@ -229,8 +229,9 @@ def add_data_format_flags(parser: argparse.ArgumentParser) -> None:
     )
 
     parser.add_argument(
+        "--chunk_shape",
         "--chunk_size",
```
Comment (jstriebel) [inline wkcuber/wkcuber/_internal/utils.py:233]: Should we also add a deprecation warning here? But also fine as-is.

**Review by jstriebel — COMMENTED** [review]
Nice refactoring! I added some comments, mostly about having the deprecated arguments as kw-only arguments.

**Comment by jstriebel** [comment]
PS: Please also add a changelog entry.

File: wkcuber/wkcuber/_internal/utils.py [hunk 2]
```diff
@@ -268,6 +269,15 @@ def parse_path(value: str) -> Path:
                 )
             },
         )
+    elif (
+        (value.startswith("webdav+http://") or value.startswith("webdav+https://"))
+        and "HTTP_BASIC_USER" in environ
+        and "HTTP_BASIC_PASSWORD" in environ
```
[the remark this answers is not in the document]
Comment (normanrz) [reply wkcuber/wkcuber/_internal/utils.py:275]: I don't think they are standard. This is in the `wkcuber`, why would we document that in the `Dataset` class? I put it in the `wkcuber` readme.

**Review by jstriebel — APPROVED** [review]
:100: :shipit:

**Pull request merged**